src/views/Browser.vue

```diff
@@ -62,10 +62,16 @@
                     </td>
                 </template>
                 <template v-slot:footer.prepend>
-                    <div v-show="searchSettings.query">
-                        Displaying {{ formatNumber(totalNumberOfExpressionExperiments) }} search results
-                    </div>
-                    <v-spacer/>
+                  <v-spacer/>
+                    <v-menu ref="codeSnippetMenu" location="">
+                      <template v-slot:activator = "{ on }">
+                        <v-btn plain v-on="on">Dataset Download Code
+                          <v-icon>mdi-chevron-up</v-icon>
+                        </v-btn>
+                      </template>
+                      <CodeSnippet :browsing-options="browsingOptions" :search-settings="searchSettings"
```

Comment:
If you add a `ref` on the `<CodeSnippet/>`, you can monitor the [height](https://developer.mozilla.org/en-US/docs/Web/API/Element/clientHeight) of the HTML element and get rid of the `resize` event.